lsp-bridge.el

```diff
@@ -1269,9 +1269,11 @@ So we build this macro to restore postion after code format."
 
 (defun lsp-bridge-not-only-blank-before-cursor ()
   "Hide completion if only blank before cursor."
-  (split-string (buffer-substring-no-properties
-                 (max (1- (point)) (line-beginning-position))
-                 (point))))
+  (not
+   (null
+    (split-string (buffer-substring-no-properties
+                    (max (1- (point)) (line-beginning-position))
+                    (point))))))
```

Comment:
Why need wrap ```(not (null ``` ?

Reply:
split-string returns a list, not a boolean value, although non-empty list will be treated as true value, I think it will be better to return a boolean value, andkeep consistent with other prediction function

You can see the return value in the log, it will be something like "(e)" if you express button e.